src/pages/[platform]/build-a-backend/data/customize-authz/configure-custom-identity-and-group-claim/index.mdx

```diff
@@ -76,3 +78,29 @@ const { errors, data: newTodo } = await client.models.Todo.create(
   // highlight-end
 );
 ```
+
+</InlineFilter>
+
+<InlineFilter filters={["flutter"]}>
+In your application, you can perform CRUD operations against the model with the `userPools` auth mode.
```

Comment:
I am not sure if we have a standard for this in the Gen 2 docs, but I would suggest we keep the text the same if possible across platforms. I think this language is generic enough to work for all platforms.

@aws-amplify/documentation-team is there an agreed upon standard?

Reply:
This comment applies to other sections

Reply:
Our `authorizationMode` enum has different names than the ones from the other platforms. Also, the swift team has already added their own `InlineFilter` in most of the data sections as well

Reply:
I see